.github/workflows/package-extension.yml

```
@@ -71,36 +67,36 @@ jobs:
           cache: npm
           cache-dependency-path: packages/vscode-extension/package-lock.json
 
-      - name: Change directory
-        run: cd packages/vscode-extension
-
       - name: Install dependencies
+        working-directory: packages/vscode-extension
         run: npm ci
 
+      - name: Prepare package ref name for artifact upload
+        id: prepare_name
+        run: |
+          ref_name=$(echo -n "${{ github.event.inputs.ref }}" | xargs | sed 's/[\\\/'"'"':<>|*?]/-/g') 
+          package_name=radon-ide-${ref_name}-${{ steps.date.outputs.date }}
+          echo "package_name=$package_name" >> $GITHUB_OUTPUT
+
       - name: Package extension
-        uses: HaaLeo/publish-vscode-extension@v1
+        working-directory: packages/vscode-extension
         id: package_extension
-        with:
-          dryRun: true
-          pat: stub
-          target: ${{ steps.set_targets.outputs.targets }}
-          preRelease: ${{ github.event.inputs.prerelease }}
-
-      - name: Sanitize ref name for artifact upload
-        id: sanitize
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          ref_name=$(echo -n "${{ github.event.inputs.ref }}" | sed 's/[\\\/'"'"':<>|*?]/-/g')
-          echo "ref_name=$ref_name" >> $GITHUB_OUTPUT
+          export OUT_PATH=/tmp/${{ steps.prepare_name.outputs.package_name }}.vsix
+          npm run vscode:package -- --out $OUT_PATH --target ${{ steps.set_targets.outputs.targets }}
```

nit: I don't think we need to export this, we don't pass it down to subprocesses
```suggestion
          out_path=/tmp/${{ steps.prepare_name.outputs.package_name }}.vsix
          npm run vscode:package -- --out $out_path --target ${{ steps.set_targets.outputs.targets }}
```